docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+   ]
   },
   {
    "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "from safeds.data.tabular.transformation import OneHotEncoder\n",
+    "from safeds.data.tabular.transformation import SimpleImputer\n",
     "\n",
-    "encoder = OneHotEncoder(column_names=\"sex\").fit(train_table)"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "execution_count": null,
-   "outputs": []
+    "simple_transformer = SimpleImputer(column_names=[\"age\",\"fare\"],strategy=SimpleImputer.Strategy.mean())\n",
+    "_, transformed_raw_data = simple_transformer.fit_and_transform(raw_data)"
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "4. Transform the training table using the fitted encoder, and create a set with the new names of the fitted `Column`s:\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "### Using `OneHotEncoder` to create an encoder and fit and transform the table\n",
+    "We use `OneHotEncoder` to transform categorical, non-numerical values into numerical representations with values of zero or one. In this example, we will transform the values of the `sex` column, so they can be used in the model to predict passenger survival.\n",
+    "- Use the `fit_and_transform` function of the `OneHotEncoder` to pass the table and the column names to be used as features for the prediction."
+   ]
   },
   {
    "cell_type": "code",
-   "source": "transformed_table = encoder.transform(train_table)",
+   "execution_count": 5,
    "metadata": {
     "collapsed": false
    },
-   "execution_count": null,
-   "outputs": []
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.transformation import OneHotEncoder\n",
+    "\n",
+    "one_hot_encoder = OneHotEncoder(column_names=[\"sex\", \"port_embarked\"])\n",
+    "transformer_one_hot_encoder, transformed_raw_data = one_hot_encoder.fit_and_transform(transformed_raw_data)"
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": "5. Mark the `survived` `Column` as the target variable to be predicted. Include some columns only as extra columns, which are completely ignored by the model:",
-   "metadata": {
-    "collapsed": false
-   }
+   "metadata": {},
+   "source": [
+    "### Reverse transforming `transformed_raw_data`"
+   ]
   },
   {
    "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "extra_names = [\"id\", \"name\", \"ticket\", \"cabin\", \"port_embarked\", \"age\", \"fare\"]\n",
-    "\n",
-    "train_tabular_dataset = transformed_table.to_tabular_dataset(\"survived\", extra_names=extra_names)"
+    "reverse_transfromed_raw_data = transformed_raw_data.inverse_transform_table(transformer_one_hot_encoder)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Satistics after imputing / removing / encoding\n",
```

Comment:
```suggestion
    "### Statistics after Imputing / Removing / Encoding\n",
```

Please proofread your tutorial once more and use title case for headings. Also shorting headings, as described in an earlier comment.